### PR TITLE
Forgot Password Form

### PR DESCRIPTION
**Pull request by narisettiashok**
Forgot password page upadated

### CODE REVIEW

File: src/pages/Auth/ForgotPasswordPage.jsx [hunk 1]
```diff
@@ -1,7 +1,61 @@
...
 function ForgotPasswordPage() {
     return (
         <>
-            <h4 className="w-full text-center">Forgot Password Page</h4>
+             {/* Construction Image */}
+             <div className="w-full lg:w-3/6 px-4 hidden md:block">
+                <img src={require("../../assets/images/construction.svg").default} alt="Construction"></img>
+            </div>
+
+            {/* Login Form */}
```
Comment (deepakrajendra) [inline src/pages/Auth/ForgotPasswordPage.jsx]: Correct the comment

File: src/pages/Auth/ForgotPasswordPage.jsx [hunk 2]
```diff
@@ -1,7 +1,61 @@
...
+                                id="companyName"
+                                className="form-control"
+                                placeholder="Company Name"
+                            />
+                        </div>
+
+                        <div className="text-center mt-6">
+                            <Link className="bg-blue-800 text-white active:bg-blue-600 text-sm font-bold uppercase px-6 py-3 rounded shadow hover:shadow-lg outline-none focus:outline-none mr-1 mb-1 w-full ease-linear transition-all duration-150"
+                                type="button" to="/auth/reset-password">
+                                Reset Password?
```
Comment (deepakrajendra) [inline src/pages/Auth/ForgotPasswordPage.jsx]: Remove '?'

File: src/pages/Auth/ForgotPasswordPage.jsx [hunk 3]
```diff
@@ -1,7 +1,61 @@
...
+                                Email
+                            </label>
+                            <input type="email"
+                                id="email"
+                                className="form-control"
+                                placeholder="Email"
+                            />
+                        </div>
+
+                        <div className="form-group">
```
Comment (deepakrajendra) [inline src/pages/Auth/ForgotPasswordPage.jsx]: Company name field is not required. Email alone is enough

File: src/pages/Auth/ForgotPasswordPage.jsx [hunk 4]
```diff
@@ -1,7 +1,61 @@
...
+            {/* Login Form */}
+            <div className="w-full lg:w-2/6">
+                <FormContainer>
+                    <form>
+                        <div className="form-group">
+                            <h1 className="uppercase font-bold text-xl text-center pb-3">
+                                Forgot Password?
+                            </h1>
+                            <p className="text-xs text-center pb-3">
+                                Please enter your registered email ID and company name to reset password?
```
Comment (deepakrajendra) [inline src/pages/Auth/ForgotPasswordPage.jsx]: Remove '?'

File: src/pages/Auth/ForgotPasswordPage.jsx (hunk 1 above)
Comment (narisettiashok) [reply src/pages/Auth/ForgotPasswordPage.jsx:13]: Ok

File: src/pages/Auth/ForgotPasswordPage.jsx (hunk 2 above)
Comment (narisettiashok) [reply src/pages/Auth/ForgotPasswordPage.jsx]: Resolved

File: src/pages/Auth/ForgotPasswordPage.jsx (hunk 3 above)
Comment (narisettiashok) [reply src/pages/Auth/ForgotPasswordPage.jsx]: Resolved

File: src/pages/Auth/ForgotPasswordPage.jsx (hunk 4 above)
Comment (narisettiashok) [reply src/pages/Auth/ForgotPasswordPage.jsx]: Resolved

File: src/pages/Auth/ForgotPasswordPage.jsx (hunk 1 above)
Comment (deepakrajendra) [reply src/pages/Auth/ForgotPasswordPage.jsx]: Change the comment to Forgot Password Form

Comment (narisettiashok) [reply src/pages/Auth/ForgotPasswordPage.jsx]: Resolved

Comment (deepakrajendra) [reply src/pages/Auth/ForgotPasswordPage.jsx]: I have commented on line 13. Meaning - Something has to be changed on line 13. 
'Login Form' has to be changed to 'Forgot Password Form'.

Comment (narisettiashok) [reply src/pages/Auth/ForgotPasswordPage.jsx]: Resolved

**Pull request merged**